README.md

```diff
@@ -4,15 +4,41 @@
 Repository of [Cyclus](https://fuelcycle.org/) archetypes to couple Cyclus with [OpenMC](https://docs.openmc.org/en/develop/pythonapi/generated/openmc.run.html)
 
 ## Installation 
-To install this archetype library run ``pip install .``. 
-To run tests: ``pytest`` from the main directory.
+
 
 ### Dependencies
 You will need to have [Cyclus](www.github.com/cyclus/cyclus), [OpenMC](https://docs.openmc.org).
 and their required dependencies. It is recommended to install Cyclus from source,
 then install OpenMC in a separate conda environment as their python dependencies 
 clash when both are installed via conda in the same environment.
 
+conda install -y python=3.7 cyclus cycamore hdf5 coincbc=2.9 gettext jinja2 libxml2 libxmlpp nose pytest pcre websockets xz libgfortran4 cython matplotlib notebook nb_conda_kernels pandas requests entrypoints pyyaml vtk coverage pytest-cov colorama gcc_linux-64=12.2 gxx_linux-64=12.2 libpng cmake make
+
+HDF5_DIR=$CONDA_PREFIX \
+pip install --upgrade-strategy only-if-needed --no-binary=h5py h5py
+
+
+git clone --recurse-submodules https://github.com/openmc-dev/openmc.git
+
+cd openmc/
+
+mkdir build && cd build
+
+cmake ..
+
+make
+
+sudo make install
+
+cd ../
+
+pip install .
+
+
+### Install OpenMCyclus
+To install this archetype library run ``pip install .``. 
+To run tests: ``pytest`` from the main directory.
+
```

Comment:
```suggestion

### Testing
In progress.
```
Should there be a testing subsubsection?

Reply:
All it would contain would be the sentence about running `pytest`.